packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -77,8 +71,8 @@ export const WebPropertyDetailPage = (): JSX.Element => {
   const spaPropertyKeys = Object.keys(spaProperties.data || {});
   const isSpaPropertyListEmpty = spaPropertyKeys.length === 0;
   const webPropertiesKeys = Object.keys(webProperties.data || {});
-  const isWebPropertiesListEmpty = webPropertiesKeys.length === 0;
-
+  const countOfSpas = useGetSPAPropGroupByName(propertyIdentifier, '');
```

Comment:
@akhilmhdh does this return isEmpty? 
@nikhitamore can you check?

Reply:
Other than this everything else looks good to me

Reply:
Processing... 😅  can u please rephrase that..